h2o-webserver-iface/src/main/java/water/webserver/iface/LoginType.java

```diff
@@ -0,0 +1,22 @@
+package water.webserver.iface;
+
+/**
+ * Supported login methods
+ */
+public enum LoginType {
```

Comment:
We have tests for that. Can you run these tests against this branch please @michal-raska ?

Reply:
http://mr-0xc1:8080/view/H2O-3/job/h2o-3-hadoop-smoke-pipeline/job/pk%252Fisolate-jetty8/1/